README.md

```diff
@@ -37,3 +37,22 @@ bugfix, and as a new GitHub issue otherwise.
 ## License
 MiniModelarDB is licensed under version 2.0 of the Apache License and a copy of the
 license is bundled with the program.
+
+## Development
+All code should follow the official [Rust Style Guide](https://github.com/rust-dev-tools/fmt-rfcs/blob/master/guide/guide.md). 
+Subjects not covered in the style guide, or requirements specific to this repository, are covered here. 
+
+### Documentation
+All public functions should have an accompanying doc comment that describes the purpose of the function. If necessary, 
+the doc comment can also include a description of each argument, the return value, and examples.
+
+All modules should have an accompanying doc comment that describes the general functionality of the module. A brief description 
+of the public functions, structs, enums, or other central elements of the module can be included.
+
+All private functions should have an accompanying comment that describes the purpose of the function.
+
+### Packages
```

Comment:
To be consistent with the terminology used in the Rust community the term [crate](https://crates.io/) should probably be used instead of [package](https://doc.rust-lang.org/book/ch07-01-packages-and-crates.html) in this section as [tracing-log](https://crates.io/crates/tracing-log) is a crate and not a package.